tests/e2e/tests/commands/new/new-routing.ts

```diff
@@ -0,0 +1,61 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import {ng} from '../../../utils/process';
+import {deleteDir} from '../../../utils/fs';
+
+export default function() {
+  const parentTestProjectDir = process.cwd();
+  const ngNewProjectName = 'new-test-project';
+
+  return Promise.resolve()
```

Comment:
You keep showing me awesome gems. I didn't realise the `getGlobalVariable()` before!

Will update tomorrow or today if I have time.